test/aws-sig-v4-test-suite/aws-sig-v4-test-suite/get-utf8/get-utf8.authz

```diff
@@ -1 +1 @@
-AWS4-HMAC-SHA256 Credential=AKIDEXAMPLE/20150830/us-east-1/service/aws4_request, SignedHeaders=host;x-amz-date, Signature=8318018e0b0f223aa2bbf98705b62bb787dc9c0e678f255a891fd03141be5d85
\ No newline at end of file
+AWS4-HMAC-SHA256 Credential=AKIDEXAMPLE/20150830/us-east-1/service/aws4_request, SignedHeaders=host;x-amz-date, Signature=697b34846207a3f72246f99d74ae1ee4fe54f44bb06730c58a0d339eb079596d
```

Comment:
Why has this been changed? These were copies of the files that were officially published by AWS. Not sure if they should be altered.

Reply:
Any chance you can point me to the source of these test files?

Reply:
This was the link:
https://docs.aws.amazon.com/general/latest/gr/signature-v4-test-suite.html

but it looks like AWS has restructured that part of the docs.
Need to have a look where they put it now.